cowamm/profitability/lp_prices/cow_amm_lp_4420687.sql

```diff
@@ -0,0 +1,105 @@
+-- Issues with the other version (4420687)
```

Comment:
What is the other version? This query references the one we are using here? Is this comment cleaned up?

Also there is no high level description of what this query does at the top, which we should do for all queries.